dags/hivemind_etl_helpers/tests/integration/test_load_envs.py

```diff
@@ -15,10 +15,10 @@ def test_mongo_envs_check_type(self):
     def test_mongo_envs_values(self):
         mongo_creds = load_mongo_credentials()
 
-        self.assertNotEqual(mongo_creds["user"], None)
-        self.assertNotEqual(mongo_creds["password"], None)
-        self.assertNotEqual(mongo_creds["host"], None)
-        self.assertNotEqual(mongo_creds["port"], None)
+        self.assertTrue(mongo_creds["user"])
```

Comment:
The mongo_creds will always have string values. You can update your assertions to be `self.assertNotEqual(variable, "")`

Reply:
Might be the case that we need to use `cache=RedisCache.from_redis_client` instead of `cache=RedisCache.from_host_and_port` as we are currently not utilizing `password`

Reply:
Great point. Could you modify the changes to use the password too?